### ISSUE TITLE
Reload speed of weapons need to be quicker/slower relative to the weapon 

### ISSUE THREAD

**Issue by joshhartford** [api-Hypernova/AC2-cube2#53]
I noticed all weapons take the same amount of time to reload, which is kinda sus. xD Pistol and minigun take the same amount of time. I think the more powerful the weapon = slower reload time?

**Comment by api-Hypernova** [comment]
Yeah if you remember that code block in game.h of settings for all the weapons, we should add another field there called 'reloadtime' or something, then when we reload we should have it just set the delay depending on the gun. Also super easy to do - give it a shot! :)

**Comment by api-Hypernova** [comment]
By the way all the "timings" in the code are milliseconds. I think the current reload is 1.5 or 2 seconds (so 1500 or 2000 ms). If you search the code for "reload" you should find it.
